src/components/ReleaseService/ReleasePlan/TriggerRelease/TriggerReleaseFormPage.tsx

```diff
@@ -62,24 +71,44 @@ export const TriggerReleaseFormPage: React.FC = () => {
     navigate(RELEASE_SERVICE_PATH.createPath({ workspaceName: namespace }));
   };
 
-  const initialValues: TriggerReleaseFormValues = {
-    releasePlan: releasePlanName,
-    snapshot: '',
-    synopsis: '',
-    description: '',
-    topic: '',
-    references: [],
-    labels: [{ key: '', value: '' }],
-  };
+  // Check if the releasePlanName corresponds to an actual release plan
+  // Don't use the releasePlanName if it's our placeholder value 'new' or if it doesn't exist
```

Comment:
hmm, I didn't get this part of the comment "if it's our placeholder value 'new'" 🤔